src/sections/header.liquid

```diff
@@ -78,26 +80,27 @@
   </header>
 
   <nav role="navigation">
-    <ul class="site-nav">
-      {% for link in linklists[section.settings.main_linklist].links %}
-        {%- assign child_list_handle = link.title | handleize -%}
-        {% if menus[child_list_handle].links != blank %}
-          <li class="site-nav--has-submenu{% if link.active %} site-nav--active{% endif %}">
-            <a href="{{ link.url }}" class="site-nav__link">
+    <ul class="menu">
+      {% for link in linklists[section.settings.menu].links %}
+        {% if link.links != blank %}
+          <li class="menu-item{% if link.active %} menu-item--active{% endif %}">
+            <a href="{{ link.url }}" class="menu-link">
```

Comment:
I was wondering if do we need all CSS classes in this file? Because I can't see a reference in our stylesheets.

menu-link, menu-item, menu-item--active, sub-menu-item...

Reply:
I was thinking the same thing but I'm not sure how to approach this. I was ready to remove all classes but wasn't entirely convinced 🤷‍♂️.

Reply:
I'd say the biggest benefit in keeping these classes is having Liquid logic already implemented for for the developer for, say, the `link.active` state.

Having said that, I'm leaning towards removing the classes altogether. I feel like doing so would maintain consistency across all sections, i.e. not adding classes inside the markup that we don't have intention of styling.

Reply:
Totally agree what you said. I'm leaving the option to you 😀